jobs/consul-dns-setup/spec

```diff
@@ -0,0 +1,9 @@
+---
+name: consul-dns-setup 
```

Comment:
Rename this job to "consul-gorouter-dns-setup"

Reply:
Will change job same as suggested.